components/transaction/TransactionDetails.tsx

```diff
@@ -121,9 +121,9 @@ function Alert(props: AlertProps) {
   return <MuiAlert elevation={6} variant="filled" {...props} />;
 }
 
-const TransactionDetails = (hash_value: String) => {
+const TransactionDetails = (value: String) => {
```

Comment:
This is a functional component. The `value` here is the `props` of the component which is will never be a `String`. You may reference to this cheatsheet how you can define the types for use in the compoent.

https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#function-components